recipes/ignition-tools/all/conanfile.py

```diff
@@ -88,18 +99,59 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.recipe_folder, "..", "source"))
 
         # Remove MS runtime files
         for dll_pattern_to_remove in ["concrt*.dll", "msvcp*.dll", "vcruntime*.dll"]:
             tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), dll_pattern_to_remove)
 
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+                tools.Version(self.version))
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file, version):
+        content = textwrap.dedent("""\
+            set(IGN_TOOLS_CONAN_PACKAGE_DIR ${{CMAKE_CURRENT_LIST_DIR/../..}})
+            list(APPEND IGNITION-TOOLS_BINARY_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/bin)
+            list(APPEND IGNITION-TOOLS_INCLUDE_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/include)
+            list(APPEND IGNITION-TOOLS_LIBRARY_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/libs)
+            list(APPEND IGNITION-TOOLS_CFLAGS -I/${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/include)
+            list(APPEND IGNITION-TOOLS_CXX_FLAGS -std=c++11)
+            set(ignition-tools{major}_VERSION_MAJOR {major})
+            set(ignition-tools{major}_VERSION_MINOR {minor})
+            set(ignition-tools{major}_VERSION_PATCH {patch})
+            set(ignition-tools{major}_VERSION_STRING "{major}.{minor}.{patch}")
+        """.format(major=version.major, minor=version.minor, patch=version.patch))
+        tools.save(module_file, content)
+
     def package_info(self):
-        version_major = tools.Version(self.version).major
-        self.cpp_info.names["cmake_find_package"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.names["cmake_find_package_multi"] = "ignition-tools{}".format(version_major)
-
-        self.cpp_info.components["libignition-tools"].libs = ["ignition-tools-backward"]
-        self.cpp_info.components["libignition-tools"].includedirs.append("include/ignition/tools{}".format(version_major))
-        self.cpp_info.components["libignition-tools"].names["cmake_find_package"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.components["libignition-tools"].names["cmake_find_package_multi"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.components["libignition-tools"].names["pkg_config"] = "ignition-tools{}".format(version_major)
+        lib_name = "ignition-tools"
+        self.cpp_info.names["cmake_find_package"] = lib_name
+        self.cpp_info.names["cmake_find_package_multi"] = lib_name
+        self.cpp_info.names["cmake_paths"] = lib_name
```

Comment:
AFAIK the `cmake_paths` generator always outputs `conan_paths.cmake` this should not be required

```suggestion
```

Reply:
I did not really know about this. Thanks for pointing out!